src/scss/_responsive-flat.scss

```diff
@@ -5,7 +5,6 @@
 
 /// Make a table 'ResponsiveFlat'
 @mixin oTableResponsiveFlat {
-	display: block;
```

Comment:
The understanding is that `float: left` applies `display: block` automatically. This is something I learnt from @onishiweb !